client/post-editor/style.scss

```diff
@@ -105,6 +105,10 @@
 	}
 }
 
+.post-editor__drafts-button.drafts-button {
```

Comment:
:)

I think it's also more scalable to target children with descendants instead of double classes. Less room for weird interactions down the road.